CHANGELOG.rst

```diff
@@ -37,6 +37,9 @@ Removed
 ~~~~~~~~
 * Removed check-licence script (cleanup) #5092
   Contributed by @kroustou
+* Updated Travis CI and CircleCI builds to only use Python 3 (cleanup) #5090
```

Comment:
I think the other significant change to mention is that py2 was used in the Makefile. Because other scripts and systems may rely on it it worth highlighting.

Maybe something along the lines?

```suggestion
* Updated Makefile and build system to only use Python 3 (cleanup) #5090
```